### PR TITLE
Sandblast Context Fixes

### PR DESCRIPTION
**Pull request by amshamah419**
Addresses demisto/etc#14001

### CODE REVIEW

File: Integrations/integration-Sandblast.yml [hunk 1]
```diff
@@ -242,7 +242,27 @@ script:
                     maliciousData[uniqueKeys[key].toUpperCase()] = raw.Contents.response[uniqueKeys[key]];
                 }
             }
+            setContext('DBotScore', {'Indicator': raw.Contents.response.md5, 'Type': 'hash', 'Vendor': 'SandBlast', 'Score': 3, 'SubType': 'av'});
```
Comment (yuvalbenshalom) [inline Integrations/integration-Sandblast.yml]: why do you use setContext? I don't think we should use it any more, but rather return the context values in the entry

File: Integrations/integration-Sandblast.yml [hunk 2]
```diff
@@ -242,7 +242,27 @@ script:
...
+                    Severity: raw.Contents.response.te.severity
+                }
+            };
+            for (var key in uniqueKeys) {
+                if (raw.Contents.response[uniqueKeys[key]]) {
+                    maliciousData[uniqueKeys[key].toUpperCase()] = raw.Contents.response[uniqueKeys[key]];
+                }
+            }
+
+            setContext('DBotScore', {'Indicator': raw.Contents.response.md5, 'Type': 'hash', 'Vendor': 'SandBlast', 'Score': 3, 'Subtype':'te'});
```
Comment (yuvalbenshalom) [inline Integrations/integration-Sandblast.yml]: same here with setContext

File: Integrations/integration-Sandblast.yml [hunk 3]
```diff
@@ -251,7 +271,10 @@ script:
                     ec['File(val.' + uniqueKeys[key].toUpperCase() + ' == obj.' + uniqueKeys[key].toUpperCase() + ')'] = val;
                 }
             }
+            setContext('DBotScore', {'Indicator': raw.Contents.response.md5, 'Type': 'hash', 'Vendor': 'SandBlast', 'Score': 3, 'SubType':'--'});
```
Comment (yuvalbenshalom) [inline Integrations/integration-Sandblast.yml]: same here with setContext

File: Integrations/integration-Sandblast.yml [hunk 4]
```diff
@@ -222,63 +222,96 @@ script:
     if (command === 'sandblast-upload' || command === 'sandblast-query' || command === 'sb-upload' || command === 'sb-query') {
         var uniqueKeys = ['md5', 'sha1', 'sha256'];
```
Comment (yuvalbenshalom) [inline Integrations/integration-Sandblast.yml]: change this to hashTypes

File: Integrations/integration-Sandblast.yml [hunk 5]
```diff
@@ -222,63 +222,96 @@ script:
...
                 }
             };
             for (var key in uniqueKeys) {
-                if (raw.Contents.response[uniqueKeys[key]]) {
-                    maliciousData[uniqueKeys[key].toUpperCase()] = raw.Contents.response[uniqueKeys[key]];
+                if (contents.response[uniqueKeys[key]]) {
+                    maliciousData[uniqueKeys[key].toUpperCase()] = contents.response[uniqueKeys[key]];
                 }
             }
+            var dbot_con = {'Indicator': raw.Contents.response.md5, 'Type': 'hash', 'Vendor': 'SandBlast', 'Score': 3, 'SubType': 'av'};
```
Comment (yuvalbenshalom) [inline Integrations/integration-Sandblast.yml]: @mayagoldb question: are we adding file dot score only with md5? should we add it with other hashes if we have them?

Comment (mayagoldb) [reply Integrations/integration-Sandblast.yml]: @yuvalbenshalom @amshamah419 If our indicator is sha1 or sha256 we should also add dbot score

Comment (yuvalbenshalom) [reply Integrations/integration-Sandblast.yml]: @amshamah419 lets take a look at it together and see if there is a need to add sha1/256

**Pull request merged**